src/LangStrings.inc.h

```diff
@@ -455,3 +455,5 @@ DECLARE_STRING(SHIPS_DISPLAY_MODE_TOGGLE)
 DECLARE_STRING(SWITCHED_TO_ROTATIONAL)
 DECLARE_STRING(SWITCHED_TO_PARENT)
 DECLARE_STRING(L4L5_DISPLAY_MODE_TOGGLE)
+DECLARE_STRING(INCREASE_TIME_ACCELERATION)
+DECLARE_STRING(DECREASE_TIME_ACCELERATION)
```

Comment:
Done added the translations to en.json. BTW if you need it, I can also keep the Italian translation (it.json) in synch whenever I add a line to en.json